examples/FOV.py

```diff
@@ -0,0 +1,94 @@
+"""
+=========================================================================
+Using sunpy-soar to Retrieve and Plot Fields of View (FOV) on a Solar Map
+=========================================================================
+
+This example demonstrates how to fetch FOV values and Solar Orbiter data using ``sunpy.net.Fido``,
+and plot them on a ``sunpy.map.Map``.
```

Comment:
```suggestion
This example demonstrates how to fetch Solar Orbiter's Field of View (FOV) for each instrument values and Solar Orbiter data using `sunpy.net.Fido` and plot them on a `sunpy.map.Map`.
```
Do these not link correctly?